UnityProject/Assets/Scripts/Systems/Chemistry/Effects/ChemExplosion.cs

```diff
@@ -39,7 +42,7 @@ public override void Apply(MonoBehaviour sender, float amount)
 			var strength = (float)(-463+205*Mathf.Log(amount)+75*Math.PI)*potency;
 
 
-			if (insideBody && strength > 0)
+			if (insideBody && strength > 0 && !isEMP)
```

Comment:
Our coding guidelines say to use == false instead of negating. I don't like it but it is what it is